### PR TITLE
feat: add orchestrator example

### PR DESCRIPTION
**Pull request by haowen159**
resolve https://github.com/apecloud/kubeblocks-addons/issues/798

### CODE REVIEW

**Comment by JashBook** [comment]
Add README.md

**Comment by haowen159** [comment]
> Add README.md

Added via https://github.com/apecloud/kubeblocks-addons/pull/808/commits/cc71ca4e2dccd840df328d81bbf068bb688801d0

File: examples/orchestrator/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# orchestrator
+
+orchestrator is a highly available, administrative, and visual tool for managing MySQL replication topologies.
```
Comment (JashBook) [inline examples/orchestrator/README.md:3]: Orchestrator is a MySQL high availability and replication management tool, runs as a service and provides command line access, HTTP API and Web interface.

File: examples/orchestrator/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+# orchestrator
```
Comment (JashBook) [inline examples/orchestrator/README.md:1]: Orchestrator

**Pull request merged**